lib/shopify_api/session.rb

```diff
@@ -19,11 +19,11 @@ def setup(params)
         params.each { |k,value| public_send("#{k}=", value) }
       end
 
-      def temp(domain, token, &block)
-        session = new(domain, token)
+      def temp(domain, token, extra = nil, &block)
```

Comment:
Just thought to not repeat the code, but I agree with you, will fix it in next commit ... ;)